charts/apisix-ingress-controller/templates/configmap.yaml

```diff
@@ -20,7 +20,16 @@ data:
     # log options
     log_level: {{ .Values.config.logLevel | quote }}
     log_output: {{ .Values.config.logOutput | quote }}
+    cert_file: {{ .Values.config.certFile | quote }}
+    key_file: {{ .Values.config.keyFile | quote }}
     http_listen: {{ .Values.config.httpListen | quote }}
+    https_listen: {{ .Values.config.httpsListen | quote }}
+    {{- if gt (len .Values.config.kubernetes.appNamespaces) 0 }}
```

Comment:
Should be `(len .Values.config.kubernetes.ingressPublishService) 0`?

Reply:
@tokers yes, let me fix it.